paddle/fluid/platform/gpu_info.cc

```diff
@@ -100,25 +100,25 @@ size_t GpuMinChunkSize() {
 
 size_t GpuMaxChunkSize() {
   size_t total = 0;
-  size_t available = 0;
+  size_t available_memory = 0;
 
-  GpuMemoryUsage(&available, &total);
```

Comment:
The older version is better than the new version. Since this function is GpuMaxChunkSize, using `available_memory` is wordiness.

Reply:
Done, thanks